uk.co.compucorp.civicrm.hrcontactactionsmenu/CRM/HRContactActionsMenu/Component/GroupButtonItem.php

```diff
@@ -116,7 +114,7 @@ public function render() {
 
     if ($this->addBottomMargin) {
       $buttonMarkup =  sprintf('      
-        <div class="crm_contact_action_menu__bottom_margin">
+        <div class="crm_contact-actions--bottom-margin">
```

Comment:
what's this class for? Please add screenshot

Reply:
This class adds bottom margin of `10px` to the element it is used in. For example here:
<img width="272" alt="screen shot 2018-01-24 at 1 21 02 pm" src="https://user-images.githubusercontent.com/6307362/35321104-3fb37846-010e-11e8-9c0c-25cba432aaa7.png">

the class is used to add 10px space between these two button.

Reply:
this is a layout rule, it should be handled by the parent. The parent always dictates the layout of its children. So please get rid of that class

Reply:
Done! refactored!